src/main/kotlin/no/skatteetaten/aurora/boober/feature/FluentbitSidecarFeature.kt

```diff
@@ -342,7 +344,7 @@ fun getLogRotationExcludePattern(logFilePattern: String): String {
     return logFilePattern.replace("*", "*.[1-9]")
 }
 
-fun generateFluentBitConfig(loggerIndexes: List<LoggingConfig>, application: String, cluster: String): String {
+fun generateFluentBitConfig(defaultIndex: String, loggerIndexes: List<LoggingConfig>, application: String, cluster: String, bufferSize: Int): String {
```

Comment:
Denne funksjonen begynner å bli litt stor etter min mening. Kunne splittet det opp litt mer med extensions functions. Eksempel for inputs:
```
val inputs = loggerIndexes.generateInputs(buffferSize)
val filter = loggerIndexes.generateFilters()
```

Kunne også gått videre nedover her for å trekke ut mer av config slik at det blir enklere å resonnere over configen og hva den gjør for vi som ikke er så kjent med fluentbit. Du ser om du er enig eller ikke :)

Reply:
Da har jeg pushet en refaktorering. Vil du se om det ble bedre eller ikke?